doc/source/cluster/guide.rst

```diff
@@ -42,6 +42,22 @@ To simplify Operator configuration, Ray provides a :ref:`a Helm chart <Ray-helm>
 Installing the Helm chart will create an Operator Deployment.
 The Operator manages autoscaling Ray clusters; each Ray node runs in its own K8s Pod.
 
+Handling dependencies
+---------------------
+
+In general, Ray expects each worker to have the same environment (i.e. the same working directory, the same available Python packages, and environment variables.)
```

Comment:
What about something like "there are 2 ways to setup dependencies in Ray. (1) Prepare the same environment across the cluster (See below). But there are times that users need to specify per job/task/actor dependencies (e.g., Actor application). In this case, (2) users can use runtime environment API <link>"